docs/examples/driver_examples/Qcodes example with Rigol DS1074 Z.ipynb

```diff
@@ -0,0 +1,317 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "_ = plot_dataset(datasaver.dataset)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, the same measurement can also be performed using a helper function `expand_setpoints_helper`. This function takes a ParameterWithSetpoints and acquires the parameter along with it's setpoints. The data is returned in a format prepared to insert into the dataset. Parameter `get_trace` is a ParameterWithSetpoints. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Starting experimental run with id: 9\n"
+     ]
+    }
+   ],
+   "source": [
+    "meas_helper = Measurement()\n",
+    "meas_helper.register_parameter(rigol.channels.ch1.get_trace)\n",
+    "\n",
+    "with meas_helper.run() as datasaver:\n",
+    "        trace_val = trace()\n",
+    "        time_val = time()\n",
```

Comment:
there is no need for this, `expand_setpoints_helper` takes care of "getting" parameter values.
```suggestion
```

Reply:
Corrected